plugins/quay/README.md

```diff
@@ -24,6 +24,9 @@ This plugin will show you information about your container images within Quay re
        changeOrigin: true
        # Change to "false" in case of using self hosted quay instance with a self-signed certificate
        secure: true
+
+   quay:
```

Comment:
Can you please describe the meaning of this setting in the `README.md` as well? We may want to include it as a separate and optional configuration step. When listed as a part of the mandatory proxy setting, it may be viewed as a required step.